### ISSUE TITLE
Occasionally throws Error: ENOTEMPTY: directory not empty

### ISSUE THREAD

**Issue by tlvince** [SamVerschueren/dev-time#5]
Occasionally, clearing the Git repo fails with:

``` shell
❯ ./node_modules/.bin/dev-time kentcdodds
Error: ENOTEMPTY: directory not empty, rmdir '/var/folders/_n/vc1rw4250cbb96q5pfkflndw0000gn/T/4b50bef2-ffa6-4d65-b43d-01906829b456/.git'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at rmkidsSync (/private/tmp/node_modules/rimraf/rimraf.js:334:11)
    at rmdirSync (/private/tmp/node_modules/rimraf/rimraf.js:324:7)
    at rimrafSync (/private/tmp/node_modules/rimraf/rimraf.js:295:9)
    at /private/tmp/node_modules/rimraf/rimraf.js:332:5
    at Array.forEach (native)
    at rmkidsSync (/private/tmp/node_modules/rimraf/rimraf.js:331:26)
    at rmdirSync (/private/tmp/node_modules/rimraf/rimraf.js:324:7)
    at Function.rimrafSync [as sync] (/private/tmp/node_modules/rimraf/rimraf.js:295:9)
```

If I run that a second time, often no error is thrown; this happens with arbitrary GitHub usernames.

dev-time-cli v1.3.2 (which pulls in dev-time 0.4.0), OS X 10.10.5.

**Comment by SamVerschueren** [comment]
Seems something to do with `rimraf` which is used by `del` which is used to clean up the temporary directory. @sindresorhus @schnittstabil any ideas?

**Comment by schnittstabil** [comment]
@tlvince what files remain in the temp folder?

If it is a OS X only issue, then a Finder instance might be the cause:

> .DS_Store files are used by Mac OS X to store folder specific metadata information. They are created in every folder that Mac OS X Finder accesses, even network volumes and external devices. &lt;[http://osxdaily.com/](http://osxdaily.com/2009/12/31/what-is-a-ds_store-file/)&gt;

**Comment by tlvince** [comment]
> @tlvince what files remain in the temp folder?

Looks like the entire Git repo that dev-time clones:

```
❯ tree -a /var/folders/_n/vc1rw4250cbb96q5pfkflndw0000gn/T/4b50bef2-ffa6-4d65-b43d-01906829b456/
/var/folders/_n/vc1rw4250cbb96q5pfkflndw0000gn/T/4b50bef2-ffa6-4d65-b43d-01906829b456/
├── .git
│   └── tags
├── .gitignore
├── .travis.yml
├── LICENSE
├── README.md
├── index.js
├── package.json
├── subtitles.parser.min.js
└── test
    ├── mocha.opts
    ├── test.js
    └── test.srt

2 directories, 11 files
```

**Comment by schnittstabil** [comment]
@tlvince That's weird, `.git` directories never contain `tags` directories :confused:

**Comment by tlvince** [comment]
Ah, good catch. I have a [Git hook template](http://tbaggery.com/2011/08/08/effortless-ctags-with-git.html) that creates `tags`.

**Comment by schnittstabil** [comment]
@SamVerschueren In that case, running `('git', ['clone', '--template=""', …` should do the trick

**Comment by SamVerschueren** [comment]
@schnittstabil very nice! Thanks for the information.

**Comment by SamVerschueren** [comment]
@tlvince should be fixed in `dev-time@1.3.3` thanks to @schnittstabil
